### DISCUSSION TITLE
多个视频或图片转场如何设置每个片段的时间呢？

### DISCUSSION THREAD

**Discussion by Jiiiin** [Tencent/libpag#2181] [Q&A] [answered]
比方官方的demo：https://pag.art/file/VideoTransmations.zip 玻璃破碎转场这个pag文件，我想设置第一个图片持续3s，第二个图片持续5s有办法控制吗？
我了解了时间伸缩，没有理解具体效果，是不是时间伸缩只能给一个图层设置，然后给pagFile设置duration后就会自动把多出来的时间按照模式作用给那个图层？

**Comment by kevingpqi123** [comment, marked as answer]
时间伸缩指的是支持设置一个 pag 文件的一个区间供拉伸，如一个 pag 文件长 5s，第2 ~ 4 s为伸缩区间，当 pag 文件的时间不等于 5s 时，原 pag 文件中 2 ~ 4s 的内容是可以动态伸缩的，针对的是 pag 文件整体，不是单个的图层， 同时原文件2 ~ 4s 的内容由于受到拉伸，除填充图层自己控制外，其余内容的渲染帧率会受到影响

设置第一个图片持续3s，第二个图片持续5s
让设计师输出一个这种的 pag 文件接口，转场效果通常时长都是固定的
